worlds/khrecom/Client.py

```diff
@@ -0,0 +1,226 @@
+from __future__ import annotations
+import os
+import sys
+import asyncio
+import shutil
+import logging
+import time
+from calendar import timegm
+
+import ModuleUpdate
+ModuleUpdate.update()
+
+import Utils
+
+death_link = False
+
+item_num = 1
+
+logger = logging.getLogger("Client")
+
+if __name__ == "__main__":
+    Utils.init_logging("KHRECOMClient", exception_logger="Client")
+
+from NetUtils import NetworkItem, ClientStatus
+from CommonClient import gui_enabled, logger, get_base_parser, ClientCommandProcessor, \
+    CommonContext, server_loop
+
+
+def check_stdin() -> None:
+    if Utils.is_windows and sys.stdin:
+        print("WARNING: Console input is not routed reliably on Windows, use the GUI instead.")
+
+class KHRECOMClientCommandProcessor(ClientCommandProcessor):
+    def __init__(self, ctx):
+        super().__init__(ctx)
+
+    def _cmd_deathlink(self):
+        """Toggles Deathlink"""
+        global death_link
+        if death_link:
+            death_link = False
+            self.output(f"Death Link turned off")
+        else:
+            death_link = True
+            self.output(f"Death Link turned on")
+
+class KHRECOMContext(CommonContext):
+    command_processor: int = KHRECOMClientCommandProcessor
+    game = "Kingdom Hearts RE Chain of Memories"
+    items_handling = 0b111  # full remote
+
+    def __init__(self, server_address, password):
+        super(KHRECOMContext, self).__init__(server_address, password)
+        self.send_index: int = 0
+        self.syncing = False
+        self.awaiting_bridge = False
+        # self.game_communication_path: files go in this path to pass data between us and the actual game
+        if "localappdata" in os.environ:
+            self.game_communication_path = os.path.expandvars(r"%localappdata%/KHRECOM")
+        else:
+            self.game_communication_path = os.path.expandvars(r"$HOME/KHRECOM")
+        if not os.path.exists(self.game_communication_path):
+            os.makedirs(self.game_communication_path)
+        for root, dirs, files in os.walk(self.game_communication_path):
+            for file in files:
+                if file.find("obtain") <= -1:
+                    os.remove(root+"/"+file)
```

Comment:
Recommend using `os.path.join` and `os.path.normpath` to handle path delimiters for different OS's.